src/main/resources/lib/menu.js

```diff
@@ -15,55 +15,51 @@ var globals = {
- *   @param {String} [params.urlType=null] - Control type of URL to be generated for menu items, default is 'server', only other option is 'absolute'.
+ *   @param {String} [params.urlType=Server] - Control type of URL to be generated for menu items, default is 'server', only other option is 'absolute'.
  * @returns {Object} - The set of breadcrumb menu items (as array) and needed settings.
  */
-exports.getBreadcrumbMenu = function (params) {
-    var site = libs.portal.getSite();
-    var content = libs.portal.getContent() || site; // Fallback to site if there's no content (like in errorHandlers).
+exports.getBreadcrumbMenu = function (params = {}) {
+    let site = libs.portal.getSite();
+    let content = libs.portal.getContent() || site; // Fallback to site if there's no content (like in errorHandlers).
 
-    var breadcrumbItems = []; // Stores each menu item
-    var breadcrumbMenu = {}; // Stores the final JSON sent to Thymeleaf
+    let breadcrumbItems = []; // Stores each menu item
+    let breadcrumbMenu = {}; // Stores the final JSON sent to Thymeleaf
 
     // Safely take care of all incoming settings and set defaults, for use in current scope only
-    var settings = {
+    const settings = {
         linkActiveItem: params.linkActiveItem || false,
         showHomepage: params.showHomepage || true,
         homepageTitle: params.homepageTitle || null,
         dividerHtml: params.dividerHtml || null,
-        urlType: params.urlType || null
+        urlType: params.urlType == "absolute" ? "absolute" : "server",
     };
 
-    // We only allow 'server' or 'absolute' options for URL type.
-    if (settings.urlType) {
-        switch (settings.urlType) {
-        case 'absolute':
-            break; // Pass through
-        default:
-            settings.urlType = 'server';
-        }
-    }
-
     // Loop the entire path for current content based on the slashes. Generate one JSON item node for each item.
     // If on frontpage, skip the path-loop
     if (content._path != site._path) {
-        var fullPath = content._path;
-        var arrVars = fullPath.split("/");
-        var arrLength = arrVars.length;
-        for (var i = 1; i < arrLength - 1; i++) { // Skip first item - the site - since it is handled separately.
-            var lastVar = arrVars.pop();
-            if (lastVar != '') {
-                var curItem = libs.content.get({key: arrVars.join("/") + "/" + lastVar}); // Make sure item exists
+        let fullPath = content._path;
+        let arrVars = fullPath.split("/");
+        let arrLength = arrVars.length;
+        for (let i = 1; i < arrLength - 1; i++) {
+            // Skip first item - the site - since it is handled separately.
+            let lastVar = arrVars.pop();
+            if (lastVar != "") {
+                //Could replace with 7.1.0 exists
```

Comment:
I don't know if it would work, but perhaps use exists() in a try block, with get() in the catch block? That way hopefully you can use exists() but still have the library be backwards-compatible with XP 7.0

Reply:
I was thinking about if the exist is worth it or not. 
The content is later used, so it would become exists and then get anyway. 
Is it worth it?